recipes/ixmp/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: ixmp
+  version: {{ version }}
+
+source:
+  url: https://github.com/iiasa/ixmp/archive/v{{ version }}.tar.gz
+  sha256: 512c63bc8473e667ba8571fb7125540ce47f5245b3cca71c989f089600cfd494
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - openjdk
+    - jpype1 >=0.6.2
+    - cython
+    - numpy
+    - pandas
+    - pytest >=3.0.6
```

Comment:
Is `pytest` a `run` dependency, or is it only really required for testing. If it's the former, please remove it.